FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureBlobJobStore.cs

```diff
@@ -260,5 +267,58 @@ private async Task<T> FromBlob<T>(string blobName, CancellationToken cancellatio
                 throw new StartJobFailedException($"Parse blob file '{blobName}' failed.", ex);
             }
         }
+
+        public async Task CommitJobDataAsync(Job job, CancellationToken cancellationToken = default)
+        {
+            var stagingFolder = $"{AzureStorageConstants.StagingFolderName}/{job.Id}";
+            var directoryPairs = new List<Tuple<string, string>>();
+
+            var pathItems = await _blobContainerClient.ListPathsAsync(stagingFolder, cancellationToken);
+            foreach (var path in pathItems)
+            {
+                if (path.IsDirectory == true)
```

Comment:
Can we add some more comments here (for `ListPathsAsync`) as well?

Reply:
Thanks. Fixed